authorizations/authorization-jdbc/src/main/java/org/apache/gravitino/authorization/jdbc/JdbcSecurableObjectMappingProvider.java

```diff
@@ -0,0 +1,218 @@
+        }
+        break;
+
+      case SCHEMA:
+        convertPluginPrivileges(securableObject, databasePrivileges, tablePrivileges);
+        if (!databasePrivileges.isEmpty()) {
+          databaseObject =
+              new JdbcAuthorizationObject(securableObject.name(), null, databasePrivileges);
+          authObjects.add(databaseObject);
+        }
+
+        if (!tablePrivileges.isEmpty()) {
+          tableObject =
+              new JdbcAuthorizationObject(
+                  securableObject.name(), JdbcAuthorizationObject.ALL, tablePrivileges);
+          authObjects.add(tableObject);
+        }
+        break;
+
+      case TABLE:
+        convertPluginPrivileges(securableObject, databasePrivileges, tablePrivileges);
+        if (!tablePrivileges.isEmpty()) {
+          MetadataObject metadataObject =
+              MetadataObjects.parse(securableObject.parent(), MetadataObject.Type.SCHEMA);
+          tableObject =
+              new JdbcAuthorizationObject(
+                  metadataObject.name(), securableObject.name(), tablePrivileges);
+          authObjects.add(tableObject);
+        }
+        break;
+
+      default:
+        throw new IllegalArgumentException(
+            String.format("Don't support metadata object type %s", securableObject.type()));
+    }
+
+    return authObjects;
+  }
+
+  @Override
+  public List<AuthorizationSecurableObject> translateOwner(MetadataObject metadataObject) {
+    List<AuthorizationSecurableObject> objects = Lists.newArrayList();
+    JdbcPrivilege allPri = JdbcPrivilege.valueOf(JdbcPrivilege.Type.ALL);
+    switch (metadataObject.type()) {
+      case METALAKE:
+      case CATALOG:
+        objects.add(
+            new JdbcAuthorizationObject(
+                JdbcAuthorizationObject.ALL, null, Lists.newArrayList(allPri)));
+        objects.add(
+            new JdbcAuthorizationObject(
+                JdbcAuthorizationObject.ALL,
+                JdbcAuthorizationObject.ALL,
+                Lists.newArrayList(allPri)));
+        break;
+      case SCHEMA:
+        objects.add(
+            new JdbcAuthorizationObject(metadataObject.name(), null, Lists.newArrayList(allPri)));
+        objects.add(
+            new JdbcAuthorizationObject(
+                metadataObject.name(), JdbcAuthorizationObject.ALL, Lists.newArrayList(allPri)));
+        break;
+      case TABLE:
+        MetadataObject schema =
+            MetadataObjects.parse(metadataObject.parent(), MetadataObject.Type.SCHEMA);
+        objects.add(
+            new JdbcAuthorizationObject(
+                schema.name(), metadataObject.name(), Lists.newArrayList(allPri)));
+        break;
+      default:
+        throw new IllegalArgumentException("");
+    }
+    return objects;
+  }
+
+  @Override
+  public AuthorizationMetadataObject translateMetadataObject(MetadataObject metadataObject) {
+    throw new UnsupportedOperationException("Not supported");
+  }
+
+  private void convertPluginPrivileges(
```

Comment:
maybe function name `convertJdbcPrivileges()` is better than `convertPluginPrivileges()`?

Reply:
Renamed.